<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lw::LiveWidget</class>
 <widget class="QMainWindow" name="lw::LiveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLine">
    <property name="title">
     <string>&amp;Line</string>
    </property>
    <addaction name="actionPlotProfile"/>
    <addaction name="separator"/>
    <addaction name="actionResetSelection"/>
   </widget>
   <widget class="QMenu" name="menuRectangle">
    <property name="title">
     <string>&amp;Rectangle</string>
    </property>
    <addaction name="actionSetRoi"/>
    <addaction name="actionZStackPlot"/>
    <addaction name="separator"/>
    <addaction name="actionResetSelection"/>
   </widget>
   <addaction name="menuLine"/>
   <addaction name="menuRectangle"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockSource">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Source</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QRadioButton" name="radioSourceOriginal">
       <property name="text">
        <string>Original Sinogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioSourceReconstruction">
       <property name="text">
        <string>Reconstruction</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioSourceSinogram">
       <property name="text">
        <string>Reconstructed Sinogram</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioSourceLikelihood">
       <property name="text">
        <string>Likelihood</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioSourceGradient">
       <property name="text">
        <string>Gradient</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listSourceFiles"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboKeyword"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockView">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>177</width>
          <height>167</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkColor">
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkInvert">
           <property name="text">
            <string>Invert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkLogarithmic">
           <property name="text">
            <string>Logarithmic</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkNormalize">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Normalize</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkGrid">
           <property name="text">
            <string>Grid</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonTest">
           <property name="text">
            <string>&amp;Run Reconstruction</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonForce">
           <property name="text">
            <string>&amp;Force Step</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockProfile">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Profile</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoom"/>
   <addaction name="actionLine"/>
   <addaction name="actionRectangle"/>
   <addaction name="separator"/>
   <addaction name="actionResetView"/>
  </widget>
  <widget class="QDockWidget" name="dockRoi">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Region of Interest</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>top:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QPushButton" name="buttonRoiSync">
       <property name="text">
        <string>Sync</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>width:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QSpinBox" name="spinRoiLeft"/>
     </item>
     <item row="0" column="3" colspan="3">
      <widget class="QSpinBox" name="spinRoiTop"/>
     </item>
     <item row="3" column="5" colspan="2">
      <widget class="QLineEdit" name="lineRoiHeight">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>left:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="lineRoiWidth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="3">
      <widget class="QSpinBox" name="spinRoiBottom"/>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>right:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5" colspan="2">
      <widget class="QSpinBox" name="spinRoiRight"/>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>height:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>bottom:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="4" colspan="3">
      <widget class="QPushButton" name="buttonRoiReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockZStackSource">
   <property name="windowTitle">
    <string>z-Stack</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QListWidget" name="listZStackFiles">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockZStackPlot">
   <property name="windowTitle">
    <string>z-Stack</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
    </layout>
   </widget>
  </widget>
  <action name="actionZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/ZoomIn.png</normaloff>:/Icons/ZoomIn.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom</string>
   </property>
   <property name="toolTip">
    <string>Zoom</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/SelectLine.png</normaloff>:/Icons/SelectLine.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Line</string>
   </property>
   <property name="toolTip">
    <string>Select line</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/SelectRect.png</normaloff>:/Icons/SelectRect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rectangle</string>
   </property>
   <property name="toolTip">
    <string>Select rectangle</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionResetView">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/ViewAll.png</normaloff>:/Icons/ViewAll.png</iconset>
   </property>
   <property name="text">
    <string>Reset &amp;view</string>
   </property>
   <property name="toolTip">
    <string>Reset view</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </action>
  <action name="actionPlotProfile">
   <property name="text">
    <string>Plot &amp;profile</string>
   </property>
  </action>
  <action name="actionSetRoi">
   <property name="text">
    <string>Set &amp;region of interest</string>
   </property>
  </action>
  <action name="actionResetSelection">
   <property name="text">
    <string>Reset selection</string>
   </property>
   <property name="toolTip">
    <string>Reset selection</string>
   </property>
  </action>
  <action name="actionZStackPlot">
   <property name="text">
    <string>&amp;Z-stack plot</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
